cmd/sign.go

```diff
@@ -89,40 +88,59 @@ INFO[0001] Pushed successfully, with digest "sha256:b4936e42304c184bafc9b06dde9e
 }
 
 func (s *signCmd) run() error {
-	var cm *canonicaljson.RawMessage
+	var publicKeys intoto.PublicKeys
```

Comment:
The more I think about this, the more it looks like there is far too much logic here.
(not related to the changes introduced by this PR, the actual complexity is the same).

Reply:
Opened #83 to track this.